### PR TITLE
backwards compatible env vars, SDL_VIDEODRIVER=windib, SDL_AUDIODRIVER=pulse or dsound 

### PR DESCRIPTION
**Pull request by illume**
For https://github.com/pygame/pygame/issues/2282

Otherwise apps that use SDL_VIDEODRIVER=windib or SDL_AUDIODRIVER=pulse or dsound will fail when using SDL2.

### CODE REVIEW

File: src_py/__init__.py [hunk 1]
```diff
@@ -103,6 +103,11 @@ def warn(self):
         import pygame.cdrom
     except (ImportError, IOError):
         cdrom = MissingModule("cdrom", urgent=1)
+else:
+    if os.environ.get('SDL_VIDEODRIVER') == 'windib':
+        # windib isn't in SDL2, we don't want it to fail.
+        environ['SDL_VIDEODRIVER'] = 'windows'
+
```
Comment (MyreMylar) [inline src_py/__init__.py:110]: Should we do this here, or in `display.init()` ? I suspect there may be people who do:

```
import os
import pygame
os.environ['SDL_VIDEODRIVER'] = 'windb'

pygame.display.init()  # or regular pygame.init()
```

Which I think will work with SDL, but will not be caught by this.

Comment (illume) [reply src_py/__init__.py:110]: Good point.

Comment (illume) [reply src_py/__init__.py]: Updated it to do the business in `pygame.display.init()` instead.

**Comment by illume** [comment]
(note, I cancelled the travis build, it didn't fail).

**Review by ankith26 — APPROVED** [review]
This change looks good for me ;) 
Cheers!

**Comment by illume** [comment]
I fixed the SDL_strncasecmp, and added backwards compatible SDL_AUDIO_DRIVER names too.

**Pull request merged**